apps/climatemappedafrica/src/components/Navigation/index.js

```diff
@@ -28,7 +28,7 @@ const useStyles = makeStyles(
   }),
 );
 
-function Navigation({ variant, ...props }) {
+function Navigation({ variant = undefined, ...props }) {
```

Comment:
I think by default it is undefined.
```suggestion
function Navigation({ variant, ...props }) {
```

Reply:
Yes I do agree that by default it will implicitly be undefined, I started with moving everything from default props to the function and then decided it was too verbose and unnecessary as we can always implicitly tell how it will be, there must be a few components I must have missed when I did further refactoring...I am cross checking again to make sure I haven't missed any more instances....nice catch 👍